addons/website_event/models/event.py

```diff
@@ -26,6 +27,10 @@ class Event(models.Model):
 
     is_participating = fields.Boolean("Is Participating", compute="_compute_is_participating")
 
+    cover_properties = fields.Text(
+        'Cover Properties',
+        default='{"img_url": "", "background-color": "bg-secondary", "overlay_color": "bg-black-50"}')
```

Comment:
I see you use different keys than blog in here... could not it be the same ?
In fact, why not moving the related snippets options in website and use the same in both website_blog and website_event ? :heart: